guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<blockquote>Note: To follow the guide below, we assume that you have some basic understanding of deep learning and the convolutional neural networks (CNNs). You can learn about deep learning by going through these Stanford University's CS231n&nbsp;<a href="http://cs231n.github.io/convolutional-networks/" target="_blank">course notes</a>.</blockquote><p> </p><p>Note: To follow the guide below, we assume that you have conceptual understanding of deep learning and the convolutional neural networks (CNNs). For a detailed review of CNNs, you can read <a href="http://cs231n.github.io/convolutional-networks/" target="_blank">course notes</a> of Stanford University's course, CS231n: Convolutional Neural Networks for Visual Recognition</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done</p>